js/index.js

```diff
@@ -43,6 +44,8 @@ window.addEventListener('load', event => {
   document.querySelectorAll('.card').forEach(card => {
     card.addEventListener('click', () => {
       // TODO: write some code here
+      cards.classList.toggle('foobar')
+      cards.classList.toggle('abc', someBool)
       console.log(`Card clicked: ${card}`);
     });
```

Comment:
Por dejarte algún ejemplo más te dejo cómo se haría para que la puntuación aumente.
```suggestion
      console.log(`Card clicked: ${card}`);
      memoryGame.pairsClicked++
      const clickedPairsDiv = document.querySelector("#pairs-clicked")
      clickedPairsDiv.innerHTML = memoryGame.pairsClicked
    });
```